scrapy/utils/display.py

```diff
@@ -3,20 +3,56 @@
 """
 
 import sys
+from packaging.version import parse
+from platform import version
 from pprint import pformat as pformat_
 
 
 def _colorize(text, colorize=True):
     if not colorize or not sys.stdout.isatty():
         return text
     try:
+        import pygments.util
+        from pygments.formatters import get_formatter_by_name
         from pygments import highlight
-        from pygments.formatters import TerminalFormatter
         from pygments.lexers import PythonLexer
-        return highlight(text, PythonLexer(), TerminalFormatter())
     except ImportError:
         return text
 
+    # All Windows versions >= "10.0.14393" interpret ANSI escape sequences
+    # using terminal processing.
+    #
+    # Enable enivornment variable `ENABLE_VIRTUAL_TERMINAL_PROCESSING`
+    # to activate terminal processing.
+    if sys.platform == "win32" and parse(version()) >= parse("10.0.14393"):
+        import ctypes
+        kernel32 = ctypes.windll.kernel32
+        kernel32.SetConsoleMode(kernel32.GetStdHandle(-11), 7)
+        # set `ENABLE_VIRTUAL_TERMINAL_PROCESSING` flag
```

Comment:
Shouldn't this be before the previous line?

Reply:
yes, I have changed that.